frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1406,7 +1835,7 @@ function ReaderFooter:addToMainMenu(menu_items)
                     elseif self.settings.item_prefix == "letters" then
                         prefix_text = C_("Status bar", "Letters")
                     end
-                    return T(_("Item style: %1"), prefix_text)
+                    return T(_("Complication symbols: %1"), prefix_text)
```

Comment:
`Items`

Reply:
"Complication symbols: Icons"

Reply:
But presumably this is no longer relevant.